tests/app/views/test_audits.py

```diff
@@ -116,6 +116,38 @@ def test_should_get_audit_event_by_object(self):
         assert_equal(len(data['auditEvents']), 1)
         assert_equal(data['auditEvents'][0]['user'], 'rob')
 
+    def test_should_get_audit_event_by_missing_object(self):
```

Comment:
The name of this test seems odd - it's testing that there are no audit events for a missing object but this name sounds like it should get one.